index.html

```diff
@@ -356,8 +356,12 @@
 
   1. Let <var>new-type</var> be the <a>connection type</a> that represents the <a>underlying connection technology</a>.
   1. Let <var>new-effective-type</var> be the <a>effective connection type</a> determined by current <a>downlink</a> and <a>rtt</a> values.
-  1. Let <var>new-downlink</var> be the <a>downlink</a> value.
-  1. Let <var>new-rtt</var> be the <a>rtt</a> value.
+  1. Let <var>new-downlink</var> be the result of:
+    1. Rounding <a>downlink</a> value to nearest 25 kilobits per second.
+    1. If the resulting value is 10% greater or smaller than the value of `connection.downlink`, then set <var>new-dowlink</var> to resulting value. Otherwise, set <var>new-downlink</var> to the value of `connection.downlink`.
+  1. Let <var>new-rtt</var> be the result of:
+    1. Rounding <a>rtt</a> value to nearest 25 milliseconds.
```

Comment:
Probably want to clarify that this is as multiple of 25ms, right? like 78 would round to 75, but 90 would round to 100.